_sass/module/_referendum.scss

```diff
@@ -19,21 +7,22 @@
 
 .referendum__summary {
   h3 {
-    font-weight: $font-weight-normal;
-    color: $color-grey-2;
-    text-transform: uppercase;
-    padding-top: $spacing-base;
+    margin-top: $spacing-base * 2;
   }
 
   p {
     color: $color-grey-2;
   }
 }
 
-.referendum__region {
-  h2 {
-    text-transform: uppercase;
-    color: $color-grey-2;
+.l-section__content {
+  .uppercase {
+    font-weight: $font-weight-normal;
+  }
+}
+
+.section-header {
```

Comment:
We should probably rename this class so it is specific to the referendum page/component